### PR TITLE
Yasser - Finished the homework for Browsers week1

### PR DESCRIPTION
**Pull request by Yasserhub**

### CODE REVIEW

**Comment by Yasserhub** [comment]
Hi @robvk , sorry I have tried op donderdag but it did not work. So, I finished in Friday night.

File: 2-Browsers/Week1/homework/ex1-bookList/index.js [hunk 1]
```diff
@@ -18,7 +18,30 @@ https: //hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
...
+    bookItem.style.cssText = "display:inline-block;width:30% ; padding :15px; margin:5px; text-align:center";
+          bookList.appendChild(bookItem);
+
+    const titleAndAuthor = document.createElement('p');
+          bookItem.appendChild(titleAndAuthor);
+          titleAndAuthor.textContent = book.title + "- " + book.author;
+
+    const bookImage = document.createElement('img');
+          bookItem.appendChild(bookImage);
+          if (book.isbn === '978-0465050659') bookImage.src ='assets/the_design_of_everyday_things.jpg';
```
Comment (robvk) [inline 2-Browsers/Week1/homework/ex1-bookList/index.js:36]: This is a hardcoded way of solving the problem. Probably best to change the array of data rather than this!

File: 2-Browsers/Week1/homework/ex1-bookList/index.js [hunk 2]
```diff
@@ -18,7 +18,30 @@ https: //hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
...
 
 function createBookList(books) {
-  // TODO your code goes in here, return the ul element
+ 
+ 
+  const bookList=document.createElement('ul');
+   
+   books.forEach( book => {
+    const bookItem = document.createElement('li');
+    bookItem.style.cssText = "display:inline-block;width:30% ; padding :15px; margin:5px; text-align:center";
```
Comment (robvk) [inline 2-Browsers/Week1/homework/ex1-bookList/index.js:27]: There is a .css file, make use of it. This is very hard to follow

File: 2-Browsers/Week1/homework/ex2-aboutMe/index.js [hunk 3]
```diff
@@ -10,3 +10,17 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
...
 
 // TODO add your JavaScript code here.
+
+  document.getElementById('nickname').textContent = 'Murshed';
+  document.getElementById('fav-food').textContent = 'Mandy';
+  document.getElementById('hometown').textContent = 'Yemen';
+
+  const allLiElement= document.querySelectorAll('li');
+        allLiElement.forEach((liElement) => {
+         liElement.classList = 'list-item';
```
Comment (robvk) [inline 2-Browsers/Week1/homework/ex2-aboutMe/index.js:20]: The formatting is off here, make sure prettier runs!

File: 2-Browsers/Week1/homework/ex4-whatsTheTime/index.js [hunk 4]
```diff
@@ -8,7 +8,22 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
...
+  const today = new Date();
+  let   hour = today.getHours(),
+        minute = today.getMinutes(),
+        second = today.getSeconds();
+
+  if (minute < 10 ) minute ='0'+ minute;
+  if (hour < 10 ) hour ='0'+ hour; 
+  if (second < 10 ) second='0'+ second; 
+  const time = hour + ":" + minute + ":" + second;
+  console.log(time);
```
Comment (robvk) [inline 2-Browsers/Week1/homework/ex4-whatsTheTime/index.js:22]: Do not leave a console.log in production ready code

File: 2-Browsers/Week1/homework/ex5-catWalk/index.js [hunk 5]
```diff
@@ -21,8 +21,39 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 
    https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif
 -----------------------------------------------------------------------------*/
+const catImag=document.querySelector('img');
```
Comment (robvk) [inline 2-Browsers/Week1/homework/ex5-catWalk/index.js:24]: catImag is not a good variable name. Don't be lazy and add the extra `e` at the end ;)

File: 2-Browsers/Week1/homework/ex5-catWalk/index.js [hunk 6]
```diff
@@ -21,8 +21,39 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
...
+          
+      } else if (moveToLeft > screenCenter && moveToLeft < (screenCenter * 2) ){
+         catImag.style.left=moveToLeft+"px";
+
+      }else if(moveToLeft > (screenCenter * 2)){
+          catImag.style.left = 0 + "px";
+          moveToLeft = 0; 
+      } 
 }
+window.onload = () => setInterval(catWalk, 5000);
```
Comment (robvk) [inline 2-Browsers/Week1/homework/ex5-catWalk/index.js:58]: Making this 5000 will make it work but doesn't really solve what we would like you to solve which is that you have 2 functions that need to run at different times. Try to make this something like 100 and get it to work then. Hint: you need to know to not move if the cat is dancing, so will need to stop that interval and start it again when you need it.

File: 2-Browsers/Week1/test-reports/ex1-bookList.fail.txt [hunk 7]
```diff
@@ -0,0 +1,41 @@
+*** Unit Test Error Report ***
+
+Command failed: npx jest ex1-bookList.test.js --colors
+ FAIL  2-Browsers/Week1/unit-tests/ex1-bookList.test.js (8.565 s)
+  Generated HTML
+    ✕ HTML should be syntactically valid (231 ms)
```
Comment (robvk) [inline 2-Browsers/Week1/test-reports/ex1-bookList.fail.txt:6]: This is still failing because of this. Please fix it

File: 2-Browsers/Week1/test-reports/ex2-aboutMe.fail.txt [hunk 8]
```diff
@@ -0,0 +1,52 @@
+*** Unit Test Error Report ***
+
+Command failed: npx jest ex2-aboutMe.test.js --colors
+ FAIL  2-Browsers/Week1/unit-tests/ex2-aboutMe.test.js (6.555 s)
+  Generated HTML
+    ✓ should be syntactically valid (170 ms)
+    ✕ should have all TODO comments removed (2 ms)
+    ✕ the body font-family should be `Arial, sans-serif` (2 ms)
```
Comment (robvk) [inline 2-Browsers/Week1/test-reports/ex2-aboutMe.fail.txt:8]: There are still fails in this test. Please fix them

**Review by robvk — CHANGES_REQUESTED** [review]
Some things to work on, let me know when you have fixed them

**Comment by Yasserhub** [comment]
> Some things to work on, let me know when you have fixed them

@robvk Kindly have a look at the last update.

**Review by robvk — APPROVED** [review]
Nice!
